userprofile/sitemaps.py

```diff
@@ -1,10 +1,11 @@
-from django.contrib import sitemaps
 from django.urls.base import reverse
 
+from lib.sitemaps import AplusSitemap
 
-class UserProfileStaticViewSitemap(sitemaps.Sitemap):
-    priority = 0.1
+
+class UserProfileStaticViewSitemap(AplusSitemap):
```

Comment:
Looks like this class does not benefit from `AplusSitemap` at all. It actually makes it worse since the static priority is not an attribute anymore. Calling a method to get the static value is unnecessary. If you don't change this file at all, isn't this all the same?